docs/fast_data/configuration/single_view_creator.md

```diff
@@ -142,15 +142,27 @@ service.addHook('onClose', async() => {
 | KAFKA_CONSUMER_MAX_WAIT_TIME_MS     | -        | (v6.2.1 or higher) The maximum amount of time in milliseconds the server will block before answering the fetch request if there isn't sufficient data to immediately satisfy the requirement given by minBytes [1 byte]                                                                                                                                                                                                                                                                                                     | 500                 |
 | SV_UPDATE_VERSION                   | -        | (v6.2.1 or higher) Define which version of the `sv-update` event should be emitted by the service. Accepted values are `v1.0.0` and `v2.0.0`. By default, for retro-compatibility, version `v1.0.0` is employed                                                                                                                                                                                                                                                                                                             | v1.0.0              |
-
-If you do not want to use Kafka in the Single View Creator, you can just not set the environment variable *KAFKA_CLIENT_ID* or *KAFKA_BROKERS*. If one of them is missing, Kafka will not be configured by the service (requires *single-view-creator-lib* `v9.1.0` or higher)
+| KAFKA_SV_RETRY_TOPIC                | -        | Topic name for the [Single View Retry](#single-view-retry) mechanism. This enables the system which uses a Kafka consumer and a Kafka producer.                                                                                                                                                                                                                                                                                                                                                                             | -                   |
+| KAFKA_SV_RETRY_MAX_ATTEMPTS         | -        | Number of attempts allowed for a failed Projection Change. Defining `0` will **not** retry the failed aggregations but **will send the retry message to the DLQ**                                                                                                                                                                                                                                                                                                                                                           | 5                   |
```

Comment:
what is the DLQ here in this context? Has it been defined somewhere else? Otherwise we need a place to define what we mean as a DLQ in the Fast Data context. Otherwise we can say that the KAFKA_SV_RETRY_TOPIC is the DLQ

Reply:
You're right, I changed the DLQ keyword for topic so there's no confusion. The fact that the topic is the DLQ is specified in the sv-retry paragraph in the same page down below